src/Server.ts

```diff
@@ -0,0 +1,25 @@
+import * as dotenv from 'dotenv';
```

Comment:
Please add a header comment to all of your files to make it clear what each file accomplishes. Example:
```ts
/**Entry point for food app API, using express.js
 * This file starts the server and does xyz
 */
```
This makes it a lot easier for someone who doesn't know what each file does (i.e. me) to review and understand what's happening.